### PR TITLE
chore: remove feature flags for voice chat

### PR DESCRIPTION
**Pull request by nchamo**
The voice chat feature is live, and there is no going back. So we are removing all feature flag and config related to it

### CODE REVIEW

File: kernel/packages/config/index.ts [hunk 1]
```diff
@@ -100,10 +100,6 @@ export const RESIZE_SERVICE = qs.RESIZE_SERVICE
 export const HOTSCENES_SERVICE = qs.HOTSCENES_SERVICE
 export const REALM = qs.realm
 
-export const VOICE_CHAT_DISABLED_FLAG = location.search.includes('VOICE_CHAT_DISABLED')
```
Comment (pablitar) [inline kernel/packages/config/index.ts:103]: I'd leave this URL flag. It doesn't hurt anyone.

File: kernel/packages/shared/meta/selectors.ts [hunk 2]
```diff
@@ -33,16 +33,5 @@ export const isMOTDInitialized = (store: RootMetaState): boolean =>
...
   store.meta.config.world ? store.meta.config.world.messageOfTheDay || null : null
 
-export const isVoiceChatEnabled = (store: RootMetaState): boolean => {
-  if (!WORLD_EXPLORER || VOICE_CHAT_DISABLED_FLAG) return false
-  return !!getCommsConfig(store).voiceChatEnabled || VOICE_CHAT_ENABLED_FLAG
-}
-
-export const getVoiceChatAllowlist = (store: RootMetaState): string[] => getCommsConfig(store).voiceChatAllowlist ?? []
-
-export const isVoiceChatEnabledFor = (store: RootMetaState, userId: string): boolean =>
```
Comment (pablitar) [inline kernel/packages/shared/meta/selectors.ts:43]: I'd also leave this method. If for any reason we need to have an enable/disable functionality for voice chat in the future, we can use this as hook.

Comment (nchamo) [reply kernel/packages/shared/meta/selectors.ts:43]: Done!

File: kernel/packages/config/index.ts (hunk 1 above)
Comment (nchamo) [reply kernel/packages/config/index.ts:103]: Done!

**Review by pablitar — APPROVED** [review]
LGTM!

**Pull request merged**